goth/payment_config.py

```diff
@@ -48,6 +50,9 @@ class PaymentConfig:
     network: str
     token: str
 
+    def __post_init__(self):
+        self.platform_string = f"{self.driver}-{self.network}-{self.token.lower()}"
```

Comment:
I'm not expert in dataclasses, but shouldn't `platform_string` be also declared as a field?
```
from dataclasses import field
...
class PaymentConfig:
    ...
    platform_string: str = field(init=False)
```

I think some dataclass goodies might not work without it.

Reply:
Or maybe this should be a `@property`?

Reply:
I went with the first approach here: c6a035f